### DISCUSSION TITLE
Data consumption

### DISCUSSION THREAD

**Discussion by mikhail-gordienko** [International-Data-Spaces-Association/DataspaceConnector#72] [Q&A] [answered]
Hello!

A connector must request another connector's self-description in order to access its resources.
But the **/api/request/description** endpoint is protected. I wonder how it is supposed to work.
When I specify a target endpoint in the Configuration Manager, the connector tries to make a simple request 
without providing any credentials.

Could you please provide more details for this communication step?

connector version: 4.3.0

**Comment by juliapampus** [comment, marked as answer]
See [here](https://github.com/International-Data-Spaces-Association/DataspaceConnector/discussions/368) and please have a look at the [documentation for v4](https://international-data-spaces-association.github.io/DataspaceConnector/Faq#still-using-an-old-version) at the Communication Guide.
